wasm/validator/derive/Cargo.toml

```diff
@@ -16,4 +16,3 @@ iroha_derive_primitives.workspace = true
 syn.workspace = true
 quote.workspace = true
 proc-macro2.workspace = true
-heck = "0.4.1"
```

Comment:
Would be nice to have a check for unused dependencies in CI. I'm sure there is some tool in Rust ecosystem

Reply:
First thing from google: https://github.com/est31/cargo-udeps

Reply:
Do you think it makes sence?

Reply:
there was [this](https://github.com/hyperledger/iroha/issues/1661) issue opened long time ago

Reply:
opened a new issue